web/app/themes/mitlib-news/functions.php

```diff
@@ -19,17 +19,25 @@ function news_scripts_styles() {
 	$theme_version = wp_get_theme()->get( 'Version' );
 	$parent_version = wp_get_theme()->parent()->get( 'Version' );
 
-	// Load FontAwesome via CDN.
+	// Register stylesheets.
 	wp_register_style( 'font-awesome', '//maxcdn.bootstrapcdn.com/font-awesome/4.6.0/css/font-awesome.min.css', array(), '4.6.0' );
-	wp_enqueue_style( 'font-awesome' );
-
-	// Load local bootstrap and mobile styles.
 	wp_register_style( 'parent-style', get_template_directory_uri() . '/style.css', array(), $parent_version );
 	wp_register_style( 'bootstrap', get_stylesheet_directory_uri() . '/css/bootstrap.css', array(), $theme_version );
 	wp_register_style( 'newsmobile', get_stylesheet_directory_uri() . '/css/newsmobile.css', array(), $theme_version );
+
+	// Register javascript libraries.
+	wp_register_script( 'bootstrap', 'https://maxcdn.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js', array( 'jquery' ), '3.3.1', false );
+	wp_register_script( 'lazyload', get_stylesheet_directory_uri() . '/js/libs/lazyload.min.js', array( 'jquery' ), $theme_version, false );
```

Comment:
I don't see this get enqueued below, but it may be happening as part of `myScripts`? i.e. this may be perfectly fine but I wanted to check in in case it was supposed to be enqueued :)

Reply:
Ah, yeah - the `lazyload` script is declared as a dependency in the next line, in the third argument. So when `myScripts` is enqueued on line 39, it ends up bringing `lazyload` along with it.